mne/label.py

```diff
@@ -621,11 +623,24 @@ def split(self, parts=2, subject=None, subjects_dir=None,
 
         Notes
         -----
-        Works by finding the label's principal eigen-axis on the spherical
-        surface, projecting all label vertex coordinates onto this axis and
-        dividing them at regular spatial intervals.
+        If using 'contiguous' split, some small fringe labels may be returned
+        that are close (but not connected) to the large components.  Also,
+        ensure that the label being split is the same resolution as the surface
+        files in `subjects_dir`.
+
+        The spatial split works by finding the label's principal eigen-axis on
+        the spherical surface, projecting all label vertex coordinates onto
+        this axis and dividing them at regular spatial intervals. The
+        'contiguous' split works by finding all connected components with
+        scipy's `connected_components` algorithm.
```

Comment:
Should I just delete all of those connected component details? The algorithm will work as long as the surface mesh and label have the same resolution (high or low).